dsm/src/main/kotlin/com/epam/dsm/util/DbUtil.kt

```diff
@@ -65,7 +65,7 @@ fun Set<Column>.tableColumns() = takeIf { it.isNotEmpty() }?.joinToString(prefix
 fun Transaction.createBinaryTable() {
     execWrapper(
         """
-             CREATE TABLE IF NOT EXISTS BINARYA ($ID_COLUMN varchar(256) not null constraint binarya_pk primary key, binarya bytea);
+             CREATE TABLE IF NOT EXISTS BINARYA ($ID_COLUMN varchar(256) not null constraint binarya_pk primary key, binarya bytea, agentkey varchar(256));
```

Comment:
I'm not sure if we enforce any limit on either _agent name_ or _build version_
So 256 characters might not be enough
Handle that case